tensorflow/python/util/nest_test.py

```diff
@@ -1218,6 +1218,16 @@ def testListToTuple(self):
         expected,
     )
 
+  def testInvalidCheckTypes(self):
+    with self.assertRaises(ValueError):
```

Comment:
This got flagged by internal tests: in graph mode, attempting to `bool()` cast a Tensor raises a specific error instead (`OperatorNotAllowedInGraphError`, a subclass of `TypeError`). I think the easiest way is to test that either ValueError os TypeError is raised.